### ISSUE TITLE
Tab logical order is wrong after click page change button at Datagrid tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1935]
URL: http://clarity-ks.surge.sh/datagrid

Reproduced Steps:
1. Go to Datagrid page
2. Click the page number or page change arrow button in the sheet.
3. Use tab order to visit the webpage.

Actual Results:
The tab logical order is wrong. It goes back to the beginning of the page after clicking the page number or page change arrow button in the sheet.

Expected Results:
The tab order should make user to browser the rest content of the webpage.

Screen shot:

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/8c1bcc25-c21d-4691-9c48-54fde10f647c)

Violation of Section 508 Standards: 
1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind and disable with hand.

NOTE: we need to know if this is an implementation issue or Clarity issue.

**Comment by gnomeontherun** [comment]
When you are focused on the last page arrow or any page number and select it, the focus leaves the datagrid because the element is removed. Is the fix the new pagination to avoid this issue or should we fix this?

**Comment by gnomeontherun** [comment]
This is stale and we've updated the focus management around the Datagrid since this time, which would cover the concerns here. Closing as addressed by our other a11y auditing.
